### PR TITLE
chore(typescript): Add package references to tsconfig.json

### PR DESCRIPTION
**Pull request by GordonSmith**
Tidy up existing tsconfig.json files.

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: demos/dashy/tsconfig.json [hunk 1]
```diff
@@ -19,6 +19,26 @@
...
+        },
+        {
+            "path": "../../packages/marshaller/"
+        },
+        {
+            "path": "../../packages/other/"
+        },
+        {
+            "path": "../../packages/util/"
+        }
```
Comment (jbrundage) [inline demos/dashy/tsconfig.json]: Does dashy not need a reference to `../../packages/chart/`?

Comment (GordonSmith) [reply demos/dashy/tsconfig.json]: Dashy has a ref to marshaller which in turn has a ref to chart

File: packages/util/.vscode/tasks.json [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-            "problemMatcher": [
-                "$tsc-watch"
-            ],
-            "group": {
-                "kind": "build",
-                "isDefault": true
-            }
-        }
-    ]
-}
```
Comment (jbrundage) [inline packages/util/.vscode/tasks.json]: Was this removal intentional? We no longer need the ability to `tsc watch` the util package?

**Comment by jbrundage** [comment]
> (not ready for review - need to give it the once over myself)

Oops... I reviewed it before I saw this comment.
